### PR TITLE
feature/ST-348-get-rid-of-default-exports

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

**Comment by pgs-gmartowski** [comment]
@wirus15 Quite big code cleaning - I've changed all imports in our library.

File: src/core/models/BinLookup.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+interface IBinLookupConfigType {
```
Comment (wirus15) [inline src/core/models/BinLookup.ts]:
```typescript
export interface IBinLookupConfigType { ... }
```

File: test/ST.spec.ts [hunk 2]
```diff
@@ -163,7 +110,7 @@ describe('ST', () => {
     // then
     it('should throw an error if newJwt is not specified', () => {
       expect(() => {
-        instance.updateJWT(null);
+        instance.UpdateJwt(null);
```
Comment (wirus15) [inline test/ST.spec.ts]: why the case in the method name changed here?

Comment (pgs-gmartowski) [reply test/ST.spec.ts]: I wanted to unify all those methods - we name them with PascalCase in `ST.ts` (unfortunately :/)

File: src/core/models/BinLookup.ts (hunk 1 above)
Comment (pgs-gmartowski) [reply src/core/models/BinLookup.ts]: ok, done; I've change the same occurrences in the other models

Comment (pgs-gmartowski) [reply src/core/models/BinLookup.ts]: I've made all requested changes in `models` folder

File: src/core/models/constants/CardinalCommerce.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const ON_CARDINAL_VALIDATED_STATUS = ['SUCCESS', 'NOACTION', 'FAILURE'];
```
Comment (wirus15) [inline src/core/models/constants/CardinalCommerce.ts]: `export const ....`

Comment (pgs-gmartowski) [reply src/core/models/constants/CardinalCommerce.ts]: ok, done,  splitted

**Pull request merged**